package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "deploy-prod": "hardhat run scripts/deploy.ts --network",
-    "deploy-test": "hardhat run scripts/deploy.ts --network",
+    "deploy-test": "hardhat run scripts/deployV1FactoryContracts.ts --network",
```

Comment:
was is it intentional to only make this change for testnet?

Reply:
added it to prod